app-frontend/src/app/components/sceneItem/sceneItem.html

```diff
@@ -19,15 +19,9 @@
     </div>
   </div>
   <div class="list-group-right">
-    <button class="btn btn-square"
-            ng-click="$ctrl.toggleSelected($event)"
-            ng-if="$ctrl.isSelectable && !$ctrl.isDisabled"
-            ng-class="{'btn-secondary': $ctrl.selectedStatus}">
-      <i ng-class="{
-                   'icon-plus': !$ctrl.selectedStatus,
-                   'icon-cross': $ctrl.selectedStatus
-                   }"></i>
-    </button>
+    <rf-toggle ng-if="$ctrl.isSelectable" model="$ctrl.selectedStatus" on-change="$ctrl.toggleSelected($event)">
```

Comment:
Oh, I opened #1777 because I didn't think it was likely to have been caused by any changes in this PR but this looks like it could have caused it, might be good to take a look.